dll/win32/shell32/lang/da-DK.rc

```diff
@@ -1,9 +1,3 @@
-/* PROJECT: ReactOS Shell Library
- * FILE:    dll/win32/shell32/lang/da-DK.rc
- * PURPOSE: Danish resource file
- * UPDATED: 01 May 2009
- */
-
```

Comment:
Why remove this? Shouldn't we eventually add the new style file headers everywhere?

Reply:
This adds unnecessary diff to the patch, please do not touch these. I usually help people fixing the headers when actual translation do come in PRs.

Reply:
I didn't touch any of those headers that were properly new-styled before, and I never do.
This one wasn't properly new-styled ever. So I am free to improve it a bit without going the whole way yet.
You can tweak those headers to your hearts desire after me in your own PR @binarymaster and I won't interfere with that, but please don't force me to make them longer than before within a PR that I do intend to port back. I do really dislike to merge back only 99% of a PR, or to add additonal stuff on top in a backport, that hasn't been done on master beforehand. So please look away just for a moment here.

Reply:
Sorry, but you clearly were not supposed to touch resource headers within the scope of the 640x480-fication of dialogs.

Reply:
I really learned something from you today binarymaster. The only way being able to work with someone like you is always doing direct pushes, and omit the PR step entirely. That is the only way to workaround your weird obsession with those headers, and being simply quicker than you.